### PR TITLE
deploy dllib 

### PR DESCRIPTION
**Pull request by Le-Zheng**
artifact: 
```
<bigdl.artifactId>bigdl-dllib-SPARK_2.4</bigdl.artifactId>
<dependency>
    <groupId>com.intel.analytics.bigdl</groupId>
    <artifactId>${bigdl.artifactId}</artifactId>
    <version>${bigdl.version}</version>
    <scope>${bigdl-scope}</scope>
</dependency>
```

### CODE REVIEW

File: scala/common/utils/src/test/scala/com/intel/analytics/bigdl/utils/EngineSpec.scala [hunk 1]
```diff
@@ -1,25 +1,25 @@
...
-// * limitations under the License.
-// */
-//
-//package com.intel.analytics.bigdl.utils
-//
-//import org.apache.spark.SparkContext
-//import org.scalatest.{BeforeAndAfter, FlatSpec, Matchers}
-//
-//class EngineSpec extends FlatSpec with Matchers with BeforeAndAfter {
+/*
```
Comment (dding3) [inline scala/common/utils/src/test/scala/com/intel/analytics/bigdl/utils/EngineSpec.scala:1]: maybe we should uncomment this file?

File: scala/dllib/src/main/scala/com/intel/analytics/bigdl/dllib/keras/KerasLayer.scala [hunk 2]
```diff
@@ -21,7 +21,7 @@ import com.intel.analytics.bigdl.dllib.nn.Graph._
 import com.intel.analytics.bigdl.dllib.nn.abstractnn.{AbstractModule, Activity}
 import com.intel.analytics.bigdl.dllib.keras.{Sequential => KSequential}
 import com.intel.analytics.bigdl.dllib.nn.{Graph, StaticGraph, Container => TContainer, Input => TInput, Sequential => TSequential}
-//import com.intel.analytics.bigdl.dllib.utils.serialization.Bigdl.{AttrValue, BigDLModule}
+// import com.intel.analytics.bigdl.dllib.utils.serialization.Bigdl.{AttrValue, BigDLModule}
```
Comment (dding3) [inline scala/dllib/src/main/scala/com/intel/analytics/bigdl/dllib/keras/KerasLayer.scala:24]: maybe we can remove this line instead?

File: scala/dllib/src/main/scala/com/intel/analytics/bigdl/dllib/keras/Topology.scala [hunk 3]
```diff
@@ -22,7 +22,7 @@ import com.intel.analytics.bigdl.dllib.nn.Graph._
 import com.intel.analytics.bigdl.dllib.nn.abstractnn.{AbstractModule, Activity}
 import com.intel.analytics.bigdl.dllib.nn.{Container, StaticGraph, Sequential => TSequential}
 import com.intel.analytics.bigdl.dllib.optim._
-//import com.intel.analytics.bigdl.dllib.utils.serialization.Bigdl.BigDLModule
+// import com.intel.analytics.bigdl.dllib.utils.serialization.Bigdl.BigDLModule
```
Comment (dding3) [inline scala/dllib/src/main/scala/com/intel/analytics/bigdl/dllib/keras/Topology.scala:25]: remove this line?

File: scala/dllib/src/main/scala/com/intel/analytics/bigdl/dllib/utils/serializer/ModuleSerializable.scala [hunk 4]
```diff
@@ -21,15 +21,15 @@ import com.intel.analytics.bigdl.dllib.nn.{Container, MklInt8Convertible}
...
+// import com.intel.analytics.bigdl.serialization.Bigdl.AttrValue.ArrayValue
 import com.intel.analytics.bigdl.serialization.Bigdl.AttrValue.ArrayValue
 import com.intel.analytics.bigdl.dllib.tensor.Tensor
 import com.intel.analytics.bigdl.dllib.tensor.TensorNumericMath.TensorNumeric
 import com.intel.analytics.bigdl.dllib.utils.{ReflectionUtils, Table}
 import com.intel.analytics.bigdl.utils.{Shape => BigDLShape}
 import com.intel.analytics.bigdl.dllib.utils.serializer.converters.{DataConverter, ShapeConverter, TensorConverter}
 import com.intel.analytics.bigdl.dllib.utils.serializer.ModuleSerializer._
-//import com.intel.analytics.bigdl.dllib.utils.serialization.Bigdl._
+// import com.intel.analytics.bigdl.dllib.utils.serialization.Bigdl._
```
Comment (dding3) [inline scala/dllib/src/main/scala/com/intel/analytics/bigdl/dllib/utils/serializer/ModuleSerializable.scala:32]: remove this line?

File: scala/dllib/src/main/scala/com/intel/analytics/bigdl/dllib/utils/serializer/Types.scala [hunk 5]
```diff
@@ -21,7 +21,7 @@ import com.intel.analytics.bigdl.dllib.nn.abstractnn.{AbstractModule, Activity}
 import com.intel.analytics.bigdl.dllib.tensor.TensorNumericMath.TensorNumeric
 import com.intel.analytics.bigdl.dllib.tensor.TensorNumericMath.TensorNumeric.{NumericBoolean, NumericChar, NumericDouble, NumericFloat, NumericInt, NumericLong, NumericString}
 import com.intel.analytics.bigdl.dllib.utils.tf.TFTensorNumeric.NumericByteString
-//import com.intel.analytics.bigdl.dllib.utils.serialization.Bigdl.BigDLModule
+// import com.intel.analytics.bigdl.dllib.utils.serialization.Bigdl.BigDLModule
```
Comment (dding3) [inline scala/dllib/src/main/scala/com/intel/analytics/bigdl/dllib/utils/serializer/Types.scala:24]: same as above

File: scala/common/utils/src/test/scala/com/intel/analytics/bigdl/utils/EngineSpec.scala (hunk 1 above)
Comment (Le-Zheng) [reply scala/common/utils/src/test/scala/com/intel/analytics/bigdl/utils/EngineSpec.scala:1]: When adding scala style check, it requires adding a header for each file.
Shall we test this ut when python ut is added?

Comment (dding3) [reply scala/common/utils/src/test/scala/com/intel/analytics/bigdl/utils/EngineSpec.scala:1]: I think this ut doesn't depend on python related code. We may enable this ut and test it now. If there is error, feel free to let me know and I will work on the fix.

**Comment by Le-Zheng** [comment]
jenkins passed: http://10.112.231.51:18080/view/BigDL-2.0/job/BigDL-PRVN-lite/1228/

**Pull request merged**